src/main/java/emu/grasscutter/command/commands/WindyCommand.java

```diff
@@ -24,10 +25,23 @@ public class WindyCommand implements CommandHandler {
     @Override
     public void execute(final Player sender, final Player targetPlayer, final List<String> args) {
         // no try{} catch{} cuz i have skill issue and it doesnt work :sob:
+        // I made try{} and catch{} now leak beta
         var config = Grasscutter.getConfig();
         String windydir = config.folderStructure.windy;
-        String path = windydir + args.get(0) + ".luac";
-        targetPlayer.sendPacket(new PacketWindSeedClientNotify(path));
-        CommandHandler.sendMessage(sender, "Sobdy!");
+        String path = windydir + "/" + args.get(0) + ".luac";
+        byte[] data;
+        ByteString byteString;
+        try {
+            data = Files.readAllBytes(new File(path).toPath());
+            byteString = ByteString.copyFrom(data);
+        } catch (FileNotFoundException e) {
+            sender.sendMessage(targetPlayer, "Missing file: " + path);
```

Comment:
Fuck, it's still sender.sendMessage... stupid copilot, in-game it'll be message from "unknown person"

Reply:
> Fuck, it's still sender.sendMessage... stupid copilot, in-game it'll be message from "unknown person"

:sob: